### PR TITLE
h265e: returned utility macro for parameters copy

### PR DESCRIPTION
**Pull request by mgonchar**

### CODE REVIEW

**Comment by onabiull** [comment]
Maybe it's time to abandon macros? I believe templates here would give safer solution.

**Comment by mgonchar** [comment]
how you suggest to implement such thing without macro? Currently c++ doesn't have reflection. The original request came from @ogolikov, she wants to see all parameters listed in a way I did in current commit

File: _studio/shared/include/mfx_utils.h [hunk 1]
```diff
@@ -131,4 +131,7 @@ template<class T> inline T AlignValue(T value, mfxU32 alignment)
 //#undef  FAILED
 //#define FAILED(hr)      (MFX_STS_TRACE(hr) < 0)
 
+#define mfxCopyField(Field)      buf_dst.Field = buf_src.Field
```
Comment (dmitry-gurulev) [inline _studio/shared/include/mfx_utils.h]: Macros should not mimic to ordinary function. Please re-name it.

Comment (mgonchar) [reply _studio/shared/include/mfx_utils.h]: @dmitry-gurulev your suggestion on name changing? caps lock?

**Pull request merged**